README.md

```diff
@@ -33,14 +33,46 @@ Deployment of a third-party solution requires a PR for a FluxCD deployment submi
 
 ## Pre-requisite (Linux/MacOS)
 
-This solution requires Flux CLI locally and Flux Controller on your Kubernetes cluster. Please follow the below steps for installing these pre-requisites :
+This solution requires Flux CLI locally and Flux Controller on your Kubernetes cluster. Flux requires access to a source repository via api and access to the kubernetes cluster you want to use for testing. Please follow the below steps for installing these pre-requisites.
+
+If you do not alreday have access to a running kubernetes cluster you can consider a local easy to deploy setup such as [k3s](https://k3s.io/) or you may choose a hosted service such as [AWS EKS](https://aws.amazon.com/eks/)
+
+Flux integrates into your running cluster and needs to find the configuration file for the cluster you want to use for testing. Flux will look in the default location, i.e. *~/.kube/config*. 
+
+Before setting up Flux make sure your configuration file points to the cluster you want to used for testing. You can use
+
+```bash
+helm ls --all-namespaces
+```
+
+for example to verify that a suitable kubeconfig file can be found and the cluster can be accessed. If no configuration is found you will get an error message indicating that *"http://localhost:8080/version"* cannot be accessed. Do not be confused by the port number. The port number for accessing the kubernetes cluster is part of the configuration file and the reported port in the error message is a default port.
+
+You can use
+
+```bash
+export KUBECONFIG=$PATH_TO_kubeconfig.yaml
+```
+
+to ensure the flux installation finds the cluster you want to use for testing.
+
+Once you have a kubernetes cluster running and the configuration file is properly setup you are ready to install flux.
+
 
 ```bash
 git clone https://github.com/aws-samples/eks-anywhere-addons.git
 cd eks-anywhere-addons
 chmod +x installFlux.sh
 ./installFlux.sh
 ```
+**Note:** In order to commit back to the project you need to create a fork in GitHub of the *eks-anywhere-addons* repository into your own project. After the fork is created clone the source code from your fork. To keep the projects connected you can add the AWS project as an upstream by adding
```

Comment:
To keep the projects connected, you can add the AWS project as an upstream by adding the below to your *.git/config* file.